doc/tools/docgram/doc_grammar.ml

```diff
@@ -493,7 +493,7 @@ let rec edit_SELF nt cur_level next_level right_assoc inner prod =
       if inner then
         Snterm nt (* first level *)
       else if i = 0 then
-        Snterm next_level
+        Snterm cur_level
```

Comment:
I wonder if this would work:
```suggestion
        (if right_assoc then Snterm next_level else Snterm cur_level)
```

Reply:
no, it really is cur_level as I just said above

Reply:
We would need to support non-associative levels if we have them. So ideally we would write:

```ocaml
(if left_assoc then Snterm cur_level else Snterm next_level)
```